.github/workflows/labeler.yml

```diff
@@ -22,3 +24,21 @@ jobs:
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
           configuration-path: .github/labeler.yml
           enable-versioned-regex: 0
+
+      - name: Set workflowId environment variable
+        id: set_workflow_id
+        run: |
+          if [[ "${{ github.event.action }}" == "labeled" ]]; then
```

Comment:
This should use the github API to find if the previous step added the label. It's a shame the `github/issue-labeler` doesn't have outputs, otherwise we'd have the information. Maybe we can add it.